client/my-sites/migrate/section-migrate.jsx

```diff
@@ -295,6 +301,13 @@ class SectionMigrate extends Component {
 						return;
 					}
 
+					/**
+					 * Request the site information until the site upgrades to Atomic
+					 */
+					if ( targetSiteSlug.indexOf( 'wpcomstaging' ) === -1 ) {
```

Comment:
This will not cover the case where the target site has a custom domain. It also feels a little fragile (checking for a "magic string").

Is there another thing we could check? Like whether the target site is atomic? Or maybe just wait and request this when the restore starts, since we know then that the site will have been upgraded.

Reply:
I don't know why I didn't choose the obvious option to check the `targetSite.options` to see if the site was Atomic :) Updated the code.

Checking when the restore starts might take a while and will probably cause some confusion. Updating it as early as possible seems better experience to me.